relay-metrics/src/aggregation.rs

```diff
@@ -1140,7 +1167,7 @@ impl Aggregator {
         I: IntoIterator<Item = Bucket>,
     {
         for bucket in buckets.into_iter() {
-            self.merge(project_key, bucket)?;
+            let _result = self.merge(project_key, bucket);
```

Comment:
I think this makes sense. Should we log to sentry or datadog when `_result` is not `Ok` though?

Reply:
i considered this but there's no information on the error type. I can make this  a simple error enum. I know that @jan-auer would have opinions on how we do error handlig

Reply:
I'd definitely not silently drop the error. An enum works for me, but not opinionated here.